app/controllers/supplejack_api/story_item_moves_controller.rb

```diff
@@ -6,6 +6,8 @@ class StoryItemMovesController < SupplejackApplicationController
 
     def create
```

Comment:
This controller can be removed in the future as sorting items logic belongs the frontend.